### PR TITLE
Updated Styles for Tabbed Panels and Codelens

### PR DESCRIPTION
**Pull request by TommyHubbard**
Updated Styles for Tabbed Panels and Codelens

### CODE REVIEW

**Comment by bnmnetp** [comment]
Can you take another look at the changes you made to activecode.js  -- some of the changes undo things I have recently changed.  In other cases I'm not sure why you made them.

I'll indicate them in the line-by-line comments

File: runestone/activecode/js/activecode.js [hunk 1]
```diff
@@ -233,8 +232,8 @@ ActiveCode.prototype.createOutput = function () {
 
 
     var lensDiv = document.createElement("div");
-    $(lensDiv).addClass("col-md-6");
-    $(lensDiv).css("display","none");
+    $(lensDiv).addClass("col-md-6-override");
```
Comment (bnmnetp) [inline runestone/activecode/js/activecode.js]: Why -override?
I don't see any difference in .css("display","none") which works fine and .css({"display":"none"})  Is there a particular reason for this change?

File: runestone/activecode/js/activecode.js [hunk 2]
```diff
@@ -357,7 +356,7 @@ ActiveCode.prototype.createGradeSummary = function () {
     var showGradeSummary = function (data, status, whatever) {
         var report = eval(data)[0];
         // check for report['message']
-        if (report) {
```
Comment (bnmnetp) [inline runestone/activecode/js/activecode.js]: This section undoes changes I made a few weeks ago and has nothing to do with styles.

File: runestone/activecode/js/activecode.js [hunk 3]
```diff
@@ -764,12 +764,7 @@ HTMLActiveCode.prototype.init = function(opts) {
 
 HTMLActiveCode.prototype.createOutput = function () {
     var outDiv = document.createElement("div");
-    $(outDiv).addClass("ac_output");
```
Comment (bnmnetp) [inline runestone/activecode/js/activecode.js]: This again looks like it reverts changes I made.

File: runestone/activecode/js/activecode.js [hunk 4]
```diff
@@ -1485,33 +1475,21 @@ ACFactory.createActiveCode = function (orig, lang, addopts) {
 }
 
 // used by web2py controller(s)
-ACFactory.addActiveCodeToDiv = function(outerdivid, acdivid, sid, initialcode, language) {
+ACFactory.addActiveCodeToDiv = function(outerdiv, acdiv, sid, initialcode, language) {
     var  thepre, newac;
-
```
Comment (bnmnetp) [inline runestone/activecode/js/activecode.js]: this group of changes was made to work around a problem with jquery and usernames that contains . or other special jquery characters.

File: runestone/.gitattributes [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
```
Comment (bnmnetp) [inline runestone/.gitattributes]: I'm very leary to accept anything that messes with end of line stuff.  I've run into so many problems with this totally mesing up a repo over the years.

**Comment by ericsonga** [comment]
We should close this since Brad made other changes that did allow the tabbed panels to grow.

**Pull request closed without merging**